src/WhatToCook.Application/Domain/Recipe.cs

```diff
@@ -1,20 +1,21 @@
-﻿
-namespace WhatToCook.Application.Domain;
+﻿namespace WhatToCook.Application.Domain;
 
 public class Recipe
 {
     public int Id { get; set; }
     public string Name { get; set; }
     public string Description { get; set; }
     public string TimeToPrepare { get; set; }
-    public List<Ingredient> Ingredients { get; set; }
+    public List<Ingredient> Ingredients { get; set; } = new();
     public Statistics Statistics { get; set; }
+    public string Image { get; set; }
+    public IEnumerable<PlanOfMeals> PlansOfMeals { get; set; }
```

Comment:
This one also might be a collection of type List that is initialized right away to avoid nullability issues

Reply:
done, will be in the next commit